app/scripts/reducers/api-version.js

```diff
@@ -0,0 +1,29 @@
+import {
+  API_VERSION,
+  API_VERSION_INFLIGHT,
+  API_VERSION_ERROR,
+  SET_API_VERSION_COMPAT_ERROR
+} from '../actions';
+
+export const initialState = {
+  api_version: '',
```

Comment:
Nitpick: I think we've been sticking with a camelCase convention for store properties